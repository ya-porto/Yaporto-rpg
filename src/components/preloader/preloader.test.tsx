import React from 'react';
import renderer from 'react-test-renderer';
import {Preloader} from './index';

describe('Preloader test', () => {
	it('Snapshot', () => {
		const tree = renderer.create(
			<Preloader />
		).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
