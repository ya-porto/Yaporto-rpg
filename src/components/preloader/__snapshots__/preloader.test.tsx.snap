// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preloader test Snapshot 1`] = `
<div
  className="preloader-container"
>
  <img
    alt="Загрузка"
    draggable="false"
    src="../../public/images/sword_preloader.png"
  />
</div>
`;
